br/pkg/lightning/importer/import.go

```diff
@@ -2227,3 +2230,18 @@ func filterColumns(columnNames []string, extendData mydump.ExtendColumnData, ign
 	}
 	return filteredColumns, extendValueDatums
 }
+
+// check store liveness of tikv client-go requires GlobalConfig to work correctly, so we need to init it,
+// else tikv will report SSL error when tls is enabled.
+// and the SSL error seems affects normal logic of newer TiKV version, and cause the error "tikv: region is unavailable"
+// during checksum.
+// todo: DM relay on lightning physical mode too, but client-go doesn't support passing TLS data as bytes,
+func initGlobalConfig(secCfg tikvconfig.Security) {
```

Comment:
Can we move it to an earlier place? I'm not sure where's the first usage of tikv client.

Reply:
https://github.com/pingcap/tidb/pull/45464/commits/0581febebfe855a50f7e6d90d3c56645dadccac9